eradiate/data/absorption_spectra.py

```diff
@@ -2,42 +2,221 @@
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [20000, 20500]
+     - [487, 500]
+   * - ``spectra-us76_u86_4-20500_21000``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [20500, 21000]
+     - [476, 487]
+   * - ``spectra-us76_u86_4-21000_21500``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [21000, 21500]
+     - [465, 476]
+   * - ``spectra-us76_u86_4-21500_22000``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [21500, 22000]
+     - [454, 465]
+   * - ``spectra-us76_u86_4-22000_22500``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [22000, 22500]
+     - [444, 454]
+   * - ``spectra-us76_u86_4-22500_23000``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [22500, 23000]
+     - [434, 444]
+   * - ``spectra-us76_u86_4-23000_23500``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [23000, 23500]
+     - [425, 434]
+   * - ``spectra-us76_u86_4-23500_24000``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [23500, 24000]
+     - [416, 425]
+   * - ``spectra-us76_u86_4-24000_24500``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [24000, 24500]
+     - [408, 416]
+   * - ``spectra-us76_u86_4-24500_25000``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [24500, 25000]
+     - [400, 408]
+   * - ``spectra-us76_u86_4-25000_25500``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [25000, 25500]
+     - [392, 400]
+   * - ``spectra-us76_u86_4-``
+     - ``eradiate-datasets_maker/scripts/spectra/acs/spectra/us76_u86_4.py``
+     - [25500, 25711]
+     - [389, 392]
 """
 
+import numpy as np
+import pathlib
 import xarray as xr
 
 from .core import DataGetter
 from ..util.presolver import PathResolver
+from ..util.units import ureg
 
 _presolver = PathResolver()
 
+_US76_U86_4_PATH = "spectra/absorption/us76_u86_4"
+_US76_U86_4_PREF = "spectra-us76_u86_4"
+
 
 class _AbsorptionGetter(DataGetter):
     PATHS = {
-        "usa_mls-narrowrange": "spectra/absorption/usa_mls/narrowrange/*.nc",
-        "usa_mls-fullrange": "spectra/absorption/usa_mls/fullrange/*.nc",
-        "us76_u86_4-fullrange": "spectra/absorption/us76_u86_4/fullrange/*.nc",
-        "test": "tests/absorption/us76_u86_4/*.nc"
+        **{"spectra-us76_u86_4-4000_25711": "spectra/absorption/us76_u86_4/spectra-us76_u86_4-4000_25711/*.nc"},
+        **{f"{_US76_U86_4_PREF}-{x}_{x + 500}": f"{_US76_U86_4_PATH}/{_US76_U86_4_PREF}-{x}_{x + 500}/*.nc" for x in np.arange(4000, 25500, 500)},
+        **{f"{_US76_U86_4_PREF}-25500_25711": f"{_US76_U86_4_PATH}/{_US76_U86_4_PREF}-25500_25711/*.nc"},
+        **{"test": "tests/absorption/us76_u86_4/*.nc"}
```

Comment:
Please format this to reduce line width to a reasonable value and remove unnecessary dict unpackings for improved legibility.